Documentation/Manual/Texinfo/chap-debugger.texi

```diff
@@ -22,66 +22,74 @@ To get up and running quickly with Debugger:
 
 @enumerate
 @item
- With Quicklisp loaded, invoke in repl:
+With Quicklisp loaded, invoke in REPL:
+
+@cl{(ql:quickload 'clim-debugger)}
 
-  @t{(ql:quickload 'clim-debugger)}
 @item
 Run simple test condition:
 
-  @t{(clim-debugger:with-debugger () (error "test"))}
+@cl{(clim-debugger:with-debugger () (error "test"))}
 @end enumerate
 
-Debugger is highly inspired by Slime and uses Swank to gain
-portability across implementations. Module is still under development
-and some details may change in the future.
+The debugger is inspired by SLIME's debugger and uses Swank to gain
+portability across implementations. The Application is still under
+development and some details may change in the future.
 
-Selecting frame with a pointer switches its details and marks it
-active. Each locale value may be inspected by selection with mouse
-pointer. Active frame is distinguished from others with red
-color. @t{Eval in frame} command evaluates expression in the active
-frame.
+Clicking frame with the mouse pointer toggles the display of its details
+and selects it. Each locale value may be inspected by selection with
+mouse pointer. The selected frame is distinguished from others with red
+color. The @command{Eval in frame} command evaluates expression in the
+selected frame.
 
 @node Keyboard shortcuts
 @subsection Keyboard shortcuts
 
 Warning: these key accelerators may change in the future.
 
-@table @samp
+@table @kbd
 @item M-p
- Mark previous frame active
+Mark previous frame active
+
 @item M-n
- Mark next frame active
+Mark next frame active
+
 @item m
- Show more frames
+Show more frames
+
 @item e
- Eval in active frame
+Eval in active frame
+
 @item TAB
- Toggle active frame details
+Toggle active frame details
+
 @item [0-9]
- Invoke nth restart
+Invoke nth restart
+
 @item q
- Quit debugger
+Quit debugger
 @end table
 
 @node Debugger API
 @subsection Debugger API
 
 @deffn {function} {debugger} condition me-or-my-encapsulation
 
-Starts debugger with supplied condition. Second argument should be
-supplied by an underlying implementation allowing to encapsulate or
+Starts debugger with @var{condition}. @var{me-or-my-encapsulation}
+should be supplied by the Lisp implementation allowing to encapsulate or
 supply different debugger for recursive debugger calls.
 @end deffn
 
 @deffn {macro} {with-debugger} options &body body
 
-Wraps the code in @t{body} to invoke clim debugger when condition is
-signalled (binds @t{*debugger-hook*} to @t{#'debugger}). Bindings are
-inherited by new threads. @t{options} are not used at the moment.
+Executes the code in @var{body} invoking the Clim debugger when an
```

Comment:
should be `CLIM` to preserve consistency

Reply:
I made macros for "CLIM" and "McCLIM" to ensure this.